src/site/content/en/animations/animations-examples/index.md

```diff
@@ -0,0 +1,346 @@
+---
+layout: post
+title: Examples of high-performance CSS animations
+authors:
+  - rachelandrew
+description: |
+  Demonstrating how high performance techniques can create complex and beautiful animations.
+date: 2020-10-20
+tags:
+  - animations
+  - performance
+---
+
+In this post find out how some popular animations found on CodePen have been created.
+THese animations all use the performant techniques discussed in other articles in this section.
+
+See [Why are some animations slow?](/animations-overview/) to learn the theory behind
+these recommendations, and the [Animations Guide](/animations-guide) for practical tips.
+
+## Wizard loading animation
```

Comment:
Is this the correct CodePen? The section heading is "wizard loading" but this animation is "3D sphere"

![image](https://user-images.githubusercontent.com/4713486/96823462-6b146480-13e1-11eb-93ef-46b04a3e623e.png)

Reply:
Perhaps it was intended to be this: https://codepen.io/Craaftx/pen/ExyYRMK

Reply:
c&p error